src/components/Ability/Ability.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+export default function Ability({name,description,manaCost}) {
+  return (
+    <tr>
```

Comment:
Try not to use the tag table for style, just when you are rendering data as a table